src/test/node/users/controllers.js

```diff
@@ -235,10 +235,7 @@ test('get created posts', t => {
   const expected = {
     status: 200,
     userKey: testUser.key,
-    // get a number of creatposts of test user
-    length: mockCreatedPosts.filter((post)=>{
-      return (post.key.indexOf(testUser.key) !== -1);
-    }).length,
+    length: 3,
```

Comment:
it means test user's created posts length.
if we get this number, we have to check userKey and duplicated itemKey in mockCreatedPosts.
I thought it's too ambiguous work in test code so just used magic number here.
I will replace it as const var from mockdata file. :)